spec/models/api/v0_1/trainee_attributes_spec.rb

```diff
@@ -220,6 +220,32 @@
               allow(AcademicCycle).to receive(:for_date)
             end
 
+            it do
+              expect(subject).to validate_inclusion_of(:training_route)
+                .in_array(Hesa::CodeSets::TrainingRoutes::MAPPING.values.excluding(TRAINING_ROUTE_ENUMS[:provider_led_postgrad]))
+            end
```

Comment:
Not sure why this context passed. The `start_year` is stubbed and therefore `nil`, should fail as the inclusion should be against the `Hesa::CodeSets::TrainingRoutes::MAPPING.values`